packages/flutter/lib/src/widgets/scrollable.dart

```diff
@@ -916,13 +916,16 @@ class ScrollableState extends State<Scrollable> with TickerProviderStateMixin, R
   void _receivedPointerSignal(PointerSignalEvent event) {
     if (event is PointerScrollEvent && _position != null) {
       if (_physics != null && !_physics!.shouldAcceptUserOffset(position)) {
+        event.respond(allowPlatformDefault: true);
```

Comment:
Similarly, can we add a comment here and below?

Reply:
Yes!